ui/src/test/java/com/hyperwallet/android/ui/repository/TransferMethodRepositoryImplTest.java

```diff
@@ -446,4 +450,80 @@ public Object answer(InvocationOnMock invocation) {
         verify(mockCallback, never()).onTransferMethodLoaded(any(HyperwalletTransferMethod.class));
         assertThat(mErrorsArgumentCaptor.getValue().getErrors(), hasItem(error));
     }
+
+    @Test
+    public void testCreateTransferMethod_createdPayPalAccountSuccessfully() {
```

Comment:
Tests look good but I've changed the naming convention (which I'd like you to follow) in PR #6 and also moved some mocks to the class level instead of method level, please update your tests